src/main/java/org/example/gymcrmsystem/dto/TraineeDto.java

```diff
@@ -0,0 +1,22 @@
+package org.example.gymcrmsystem.dto;
+
+import lombok.*;
+import org.example.gymcrmsystem.parser.Identifiable;
+
+import java.io.Serializable;
+import java.util.Date;
+
+@Data
```

Comment:
this annotation also provides us toStrign method. But we should not log personal information
https://www.baeldung.com/lombok-tostring